test/test_torch.py

```diff
@@ -6797,15 +6797,6 @@ def tmp(dtype, device):
     ('true_divide', 'tensor_with_inplace', _small_3d,
         lambda t, d: [_small_3d(t, d, has_zeros=False)], 1e-1,
         1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('pow', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1e-1, 1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
```

Comment:
Great cleanup!

Reply:
Some `__rpow__` tests are still left. I wasn't sure if I should remove them too.

Reply:
Good question! No, don't remove them. `__rpow__` is a different operator than pow.